prompts/generate-summary.txt

```diff
@@ -0,0 +1,22 @@
+You are an expert summarization agent.
+You will be given either a path or a verb in the format of a dictionary.
+
+If the chunk is a path, summarize what the path is used for in one sentence in the following manner:
+
+#START EXAMPLE#
+Fetch store details, update store settings, or delete the store.
+#END EXAMPLE#
+
+If the chunk is a verb, then summarize what can the verb do in terms of business logic
+and actual techical function in the following manner:
```

Comment:
Is there any advantage in using new lines?

Reply:
Resolved